diff --git a/android/build.gradle b/android/build.gradle
index 94e44e391dea500e2d72336ad6da44e4e5760302..fe27d7d925f5a47cf2e1471894ba9256c2b850e8 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -11,6 +11,7 @@ if (expoModulesCorePlugin.exists()) {
   applyKotlinExpoModulesCorePlugin()
   // Remove this check, but keep the contents after SDK49 support is dropped
   if (safeExtGet("expoProvidesDefaultConfig", false)) {
+    useDefaultAndroidSdkVersions()
     useExpoPublishing()
     useCoreDependencies()
   }
diff --git a/ios/UIDevice.swift b/ios/UIDevice.swift
index 94e44e391dea500e2d72336ad6da44e4e5760302..fe27d7d925f5a47cf2e1471894ba9256c2b850e8 100644
--- a/ios/UIDevice.swift
+++ b/ios/UIDevice.swift
@@ -184,7 +184,7 @@ extension UIDevice {
   // swiftlint:enable closure_body_length

   // Credit: https://github.com/developerinsider/isJailBroken/blob/master/IsJailBroken/Extension/UIDevice%2BJailBroken.swift
   var isSimulator: Bool {
-    return TARGET_OS_SIMULATOR != 0
+    return #if targetEnvironment(simulator) true #else false #endif
   }
